providers/class.two-factor-fido-u2f.php

```diff
@@ -23,12 +41,38 @@ static function get_instance() {
 	}
 
 	/**
+	 * Class constructor.
+	 *
+	 * @since 0.1-dev
+	 */
+	protected function __construct() {
```

Comment:
Even if we add it, parsing of this file will be failed at `\` of `self::$u2f = new u2flib_server\U2F( set_url_scheme( '//' . $_SERVER['HTTP_HOST'] ) );`.